client/app/bundles/course/group/pages/GroupShow/index.jsx

```diff
@@ -132,13 +132,13 @@ Category.propTypes = {
   notification: notificationShape,
 };
 
-export default connect((state) => ({
-  isFetching: state.groupsFetch.isFetching,
-  hasFetchError: state.groupsFetch.hasFetchError,
-  groupCategory: state.groupsFetch.groupCategory,
-  groups: state.groupsFetch.groups,
-  canManageCategory: state.groupsFetch.canManageCategory,
-  canManageGroups: state.groupsFetch.canManageGroups,
-  notification: state.notificationPopup,
-  isManagingGroups: state.groupsManage.isManagingGroups,
+export default connect(({ groups, notificationPopup }) => ({
+  isFetching: groups.isFetching,
+  hasFetchError: groups.hasFetchError,
+  groupCategory: groups.groupCategory,
+  groups: groups.groups,
+  canManageCategory: groups.canManageCategory,
+  canManageGroups: groups.canManageGroups,
+  notification: notificationPopup,
```

Comment:
Do we still need this considering we already have <NotificationPopup /> at the App component? Also, I think `<NotificationBar />` can be removed from other files as well?